.circleci/config.yml

```diff
@@ -346,9 +346,6 @@ jobs:
   build_docker_image:
     docker:
       - image: cimg/python:3.8
-    environment:
-      ECR_IMAGE_URL: 350138855857.dkr.ecr.eu-west-2.amazonaws.com/kedro-builder
-      AWS_REGION: eu-west-2
```

Comment:
We only need these once now so I've removed them from environment variables and just put them directly into the relevant commands below.